Networking/Networking/Mapper/ShippingLabelAddressValidationResponseMapper.swift

```diff
@@ -6,9 +6,12 @@ import Foundation
 struct ShippingLabelAddressValidationResponseMapper: Mapper {
```

Comment:
nit: maybe it's worth renaming this mapper as `ShippingLabelAddressValidationSuccessMapper`?

Reply:
Good catch, that makes sense